tests/README.md

```diff
@@ -78,6 +78,41 @@ Use ./local-integration-test.sh
   ./tests/robot.sh tests/test-cases/Group6-VIC-Machine/6-04-Create-Basic.robot
   ```
 
+## Run Docker command tests via makefile target
+
+There exist a makefile target for developers to run the docker command tests locally (not in CI environment) against a pre-deployed VCH.  This is a fast way for contributors to test their potential PR submissions against the CI tests locally before pushing the commit.  There is also another benefit gained from using the makefile target.  With the way it is setup, logs from the run are written out to vic/ci-results, even if the tests fails.  The method described above to run the tests locally with drone has the weakness that a failure in the test can sometimes result in no written logs to help debug the failure.
+
+There are a few requirements before using this makefile target.
+
+1. A VCH must be pre-deployed before calling this makefile target
+2. The makefile target relies on a script that looks for a few more secrets variable, TEST_URL_ARRAY, TEST_USERNAME, TEST_PASSWORD, and TEST_DATASTORE
+
+```
+ GITHUB_AUTOMATION_API_KEY: xxxxxxxxx
+  TEST_BUILD_IMAGE:""
+  TEST_URL_ARRAY: 192.168.218.207
+  TEST_USERNAME: root
+  TEST_PASSWORD: password
+  TEST_RESOURCE: /ha-datacenter/host/esxbox.localdomain/Resources
+  TEST_DATASTORE: datastore1
+  TEST_TIMEOUT: 120s
+  VIC_ESX_TEST_DATASTORE: /ha-datacenter/datastore/datastore1
+  VIC_ESX_TEST_URL: root:c*shc0w@192.168.218.207
+  DOMAIN=
+```
+To run these tests using the makefile target,
+
+```
+make local-ci-test secrets=test.secrets.nested dockertest=Group1-Docker-Commands/1-01-Docker-Info.robot vch=my_vch
+
+make local-ci-test secrets=test.secrets.nested dockertest=Group1-Docker-Commands vch=my_vch
+```
+
+Currently, only the Group1 tests are setup to use an existing VCH so this makefile target only works on the group 1 tests.  In the above, the first line runs a specific docker command test.  The second line runs the entire group 1 tests.  Notice, we define extra params to the *local-ci-test* target by specifying the param's name.  There are three parameters, *secrets*, *dockertest*, and *vch*.  The *vch* parameter must target an existing pre-deployed VCH.
```

Comment:
Currently only the Group1 tests are setup to use an existing VCH, so this makefile target only works on the Group1 tests.  In the above, the first line runs a specific docker command test.  The second line runs the entire Group1 tests.  Notice, we define extra params to the *local-ci-test* target by specifying the param's name.  There are three parameters, *secrets*, *dockertest*, and *vch*.  The *vch* parameter must target an existing pre-deployed VCH.